app/parse.py

```diff
@@ -21,27 +22,29 @@ class Course:
 
 
 def course_type(course_soup: BeautifulSoup) -> CourseType | list[CourseType]:
-    if course_soup.select_one(".Button_secondary__DNIuD") is not None:
+    if course_soup.find(attrs={"data-qa": "fulltime-course-more-details-button"}) is not None:
         return [CourseType.FULL_TIME, CourseType.PART_TIME]
-    return CourseType.PART_TIME
+    return [CourseType.PART_TIME]
```

Comment:
it's better to do conditions vice-versa
if course_soup.find(...): return part time
else return full_time + part time